titus-server-federation/src/test/java/com/netflix/titus/federation/service/GrpcFallbackTest.java

```diff
@@ -0,0 +1,115 @@
+package com.netflix.titus.federation.service;
+
+import com.netflix.titus.grpc.protogen.JobDescriptor;
+import com.netflix.titus.grpc.protogen.JobId;
+import com.netflix.titus.grpc.protogen.JobManagementServiceGrpc;
+import com.netflix.titus.runtime.endpoint.common.grpc.GrpcUtil;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.GrpcServerRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import rx.Observable;
+
+import java.util.Optional;
+
+import static com.netflix.titus.runtime.endpoint.common.grpc.GrpcUtil.createRequestObservable;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JobManagementServiceWithUnimplementedInterface extends JobManagementServiceGrpc.JobManagementServiceImplBase {
+    // All interface methods return UNIMPLEMENTED status.
+}
+
+class JobManagementServiceWithSlowMethods extends JobManagementServiceGrpc.JobManagementServiceImplBase {
+    public long createCount = 0;
+
+    public void createJob(com.netflix.titus.grpc.protogen.JobDescriptor request,
+                          io.grpc.stub.StreamObserver<com.netflix.titus.grpc.protogen.JobId> responseObserver) {
+        // Increment call count, but never respond to the responseObserver
+        createCount++;
+    }
+}
+
+public class GrpcFallbackTest {
+    private static final String appName = "app";
```

Comment:
minor nit: code conventions for these types of constants is `CAPITAL_UNDERSCORED`